src/components/Event/Event.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Text } from '../Text';
+import { EVENT_DATA_HOOKS, EVENT_DATA_KEYS } from './dataHooks';
+import styles from './Event.st.css';
+
+export interface EventProps {
+  time?: string;
+  title: string;
+  isTimeShown?: boolean;
+  isMultiday?: boolean;
+  isRightToLeft?: boolean;
+  isSelected?: boolean;
```

Comment:
we usually prefer using:
```suggestion
  showTime?: boolean;
  multiday?: boolean;
  selected?: boolean;
```
No need to have an `isRightToLeft` prop.
We use the `TPAComponentsConsumer`  for this.
You can see an example here: https://github.com/wix/wix-ui-tpa/blob/1f5ce19adcb485b4b8a43f044bfd1ec0b044ceda/src/components/Pagination/Pagination.tsx#L25